tokenize_rt.py

```diff
@@ -6,35 +6,65 @@
 import io
 import re
 import tokenize
+from typing import Iterable
+from typing import Iterator
+from typing import List
+from typing import Optional
+from typing import Pattern
+from typing import Sequence
+from typing import Tuple
+
+if False:
+    from typing import Text
```

Comment:
we're not testing python2 mypy, so you can just use `str`

Reply:
Hmm I did this to have consistency with `from __future__ import unicode_literals` (so any string literal will not be the `str` type) but I guess I can remove it. Do you still want the `flake8-typing-imports` dependency if I'm not taking advantage of it anymore here?

Reply:
yeah, it should still be there in case it regresses